pkgs/development/libraries/ncurses/default.nix

```
@@ -8,13 +8,14 @@
 , withCxx ? !stdenv.hostPlatform.useAndroidPrebuilt
 , mouseSupport ? false, gpm
 , unicodeSupport ? true
+, testers
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "6.4";
   pname = "ncurses" + lib.optionalString (abiVersion == "5") "-abi5-compat";
 
-  src = fetchurl {
+  src = let inherit (finalAttrs) version; in fetchurl {
     url = "https://invisible-island.net/archives/ncurses/ncurses-${version}.tar.gz";
```

```suggestion
  src = fetchurl {
    url = "https://invisible-island.net/archives/ncurses/ncurses-${finalAttrs.version}.tar.gz";
```

I am feeling I get really nitty at this point but I would personally prefer this